src/APP/pages/community/sections/JoinSYTForm.jsx

```diff
@@ -48,157 +54,178 @@ function JoinSYTForm({ closeModal }) {
+                <option value="Cyber Security">Cyber Security</option>
+                <option value="Data Analysis">Data Analysis</option>
+                <option value="Database Management">Database Management</option>
+                <option value="Machine Learning">Machine Learning</option>
+                <option value="Natural Language Processing">
+                  Natural Language Processing
+                </option>
+                <option value="Neural Networks">Neural Networks</option>
+                <option value="Software Engineering">
+                  Software Engineering
+                </option>
+                <option value="System Administration">
+                  System Administration
+                </option>
+                <option value="Robotics">Robotics</option>
+                <option value="Augmented Reality">Augmented Reality</option>
+                <option value="Virtual Reality">Virtual Reality</option>
+                <option value="Game Development">Game Development</option>
+                <option value="Internet of Things">
+                  Internet of Things(IoT)
+                </option>
+                <option value="Graphic Design">Graphic Design</option>
+
+                <option value="UI/UX Design">UI/UX Design</option>
+                <option value="Networking">Networking</option>
+                <option value="Operating Systems">Operating Systems</option>
+                <option value="Quantum Computing">Quantum Computing</option>
+                <option value="DevOps">DevOps</option>
+                <option value="Frontend Development">
+                  Frontend Development
+                </option>
+                <option value="Backend Development">Backend Development</option>
+                <option value="Full Stack Development">
+                  Full Stack Development
+                </option>
+                <option value="Embedded Systems">Embedded Systems</option>
+                <option value="Big Data">Big Data</option>
+              </select>
+            </label>
+
+            <button
+              type="submit"
+              className="text-white bg-gradient-to-b to-primary from-green-dark p-2.5 rounded-md md:mt-3"
+            >
+              {loading ? "Submitting..." : "Submit"}
+            </button>
+
+            {err && (
+              <p className="text-center text-red-500 italic font-normal">
+                {err}
+              </p>
+            )}
+          </form>
+        ) : (
           <button
-            type="submit"
-            className="text-white bg-gradient-to-b to-primary from-green-dark p-2.5 rounded-md md:mt-3"
+            onClick={handleClick}
+            className=" text-white bg-gradient-to-b to-primary from-green-dark p-2.5 rounded-md md:mt-3"
           >
-            {loading ? "Submitting..." : "Submit"}
+            Join Our Discord
           </button>
```

Comment:
This button is an extra step that the user has to go through before they join the community; which is unnecessary.

Instead, could you navigate the user to the DIscord invite tab immediately after the join SYT Form has been submitted?